src/web/components/Core/EnvironmentBanner.tsx

```diff
@@ -2,15 +2,15 @@ import { useEffect, useState } from 'react';
 
 import { EnvironmentVariable, GetEnvironmentVariables } from '../../services/environmentVariables';
 import { ApiError } from '../../utils/apiError';
-import { useAsyncError } from '../../utils/errorHandler';
```

Comment:
I renamed this because useAsyncError collided with useAsyncError from react router.